docs/machine-learning/how-to-guides/index.md

```diff
@@ -8,6 +8,33 @@ ms.date: 10/05/2018
 In the How to section of the ML.NET Guide you can find quick answers
 to common questions. In some cases, articles may be listed in multiple sections to make them easy to find.
 
+## ML.NET
+
+* [Apply feature engineering for model training on categorical data - ML.NET](https://github.com/dotnet/docs/blob/master/docs/machine-learning/how-to-guides/train-model-categorical-ml-net.md)
```

Comment:
@JRAlexander there is some inconsistency with links. In this file, they are absolute, not relative. At the same time, they are relative in TOC:
```markdown
## [How-to guides](machine-learning/how-to-guides/index.md)
### [Apply categorical feature engineering ](machine-learning/how-to-guides/train-model-categorical-ml-net.md)
### [Apply textual feature engineering ](machine-learning/how-to-guides/train-model-textual-ml-net.md)
### [Inspect intermediate pipeline data values](machine-learning/how-to-guides/inspect-intermediate-data-ml-net.md)
...
```

Reply:
Yup, should be relative. Thanks!